cerbero/packages/wix_packager.py

```diff
@@ -27,6 +27,7 @@
 from cerbero.utils import messages as m
 from cerbero.utils import shell, to_winepath, get_wix_prefix
 from cerbero.tools import strip
+from cerbero.tools import sign
```

Comment:
You can import this on the same line as the previous one:
`from cerbero.tools import strip, sign`

Reply:
Done